pkg/api/templates.go

```diff
@@ -18,15 +18,15 @@ type TemplateRequest struct {
 }
 
 type TemplateResponse struct {
-	UUID            string    `json:"uuid" readonly:"true"`
-	Name            string    `json:"name"`             // Name of the template
-	OrgID           string    `json:"org_id"`           // Organization ID of the owner
-	Description     string    `json:"description"`      // Description of the template
-	Arch            string    `json:"arch"`             // Architecture of the template
-	Version         string    `json:"version"`          // Version of the template
-	Date            time.Time `json:"date"`             // Latest date to include snapshots for
-	RepositoryUUIDS []string  `json:"repository_uuids"` // Repositories added to the template
-
+	UUID              string    `json:"uuid" readonly:"true"`
+	Name              string    `json:"name"`                // Name of the template
+	OrgID             string    `json:"org_id"`              // Organization ID of the owner
+	Description       string    `json:"description"`         // Description of the template
+	Arch              string    `json:"arch"`                // Architecture of the template
+	Version           string    `json:"version"`             // Version of the template
+	Date              time.Time `json:"date"`                // Latest date to include snapshots for
+	RepositoryUUIDS   []string  `json:"repository_uuids"`    // Repositories added to the template
+	RHSMEnvironmentId string    `json:"rhsm_environment_id"` // Environment ID used by subscription-manager & candlepin
```

Comment:
The ampersand seems to translate to `\u0026` with the generated docs. Might make sense to just replace it with "and"

Reply:
And how would you feel about capitalizing ID, so `RHSMEnvironmentID` instead?

Reply:
The "&" is rendered correctly in the ``iqe_content_sources_api/model/api_template_response.py`:

            `client_environment_id (str): Environment ID used by subscription-manager & candlepin. [optional]  # noqa: E501`

but not in ``iqe_content_sources/data/api.content_sources.content_sources_rest.spec.json``.

See my MR 415

Reply:
went ahead and changed to 'and' and changed struct member to ID